.github/workflows/update_tables.yml

```diff
@@ -0,0 +1,111 @@
+name: JetBrains IDE Release Date Fetcher
+
+on:
+  # Run this workflow on a schedule; every hour.
```

Comment:
Does this need to run every hour? Once a day should suffice imo

Reply:
Funny you should mention; I originally had it set to be daily, but did not want to wait so long when testing! :)

That being said, there is actually no harm in running it more frequently - if there are no changes, no commits will be made. The GitHub workflow takes about 30 seconds when it has to do everything (ie, fetch, generate, and commit a change). We can always throttle it back, if necessary. My hope was to keep it hourly to enable us to see the rough times when JB publishes new releases so that we can tune the cron accordingly (ie, if they typically release between 01:00 and 05:00 UTC, we can change the cron to run once or twice during that block, and not otherwise).

Let's monitor for now, and revisit this if we notice a pattern / feel the job is causing too much noise. :D

Reply:
Posted #13 - I think at this point, it's reasonable for us to only run this once a day.